CHANGELOG.rst

```diff
@@ -6,6 +6,11 @@ This document describes changes between each past release.
 1.8.1 (unreleased)
 ------------------
 
+**Bug fixes**
```

Comment:
It is not a bug it is a feature and @ametaireau was not ready to implement it that fast also it is a JSONAPI feature.